Examples/Physics_applications/laser_acceleration/inputs_rz

```diff
@@ -20,8 +20,7 @@ boundary.field_hi = pec pec
 #################################
 warpx.verbose = 1
 warpx.do_dive_cleaning = 0
-warpx.use_filter = 1
-warpx.filter_npass_each_dir = 0 1
+#warpx.use_filter = 1
```

Comment:
Wouldn't it be better to use `use_filter = 1` here, to test the new capability?

Reply:
Yes, here we want to use and set `warpx.use_filter = 1` explicitly, still needed to clean up after testing. Thank you for spotting this. Fixed in 6d87779fb2c2d345007.